core/server/helpers/asset.js

```diff
@@ -1,14 +1,25 @@
 // # Asset helper
-// Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`
+// Usage: `{{asset "css/screen.css"}}`, `{{asset "css/screen.css" ghost="true"}}`,
+// `{{asset "css/{lang}.css" lang=(lang)}}`
 //
 // Returns the path to the specified asset. The ghost flag outputs the asset path for the Ghost admin
 var proxy = require('./proxy'),
     _ = require('lodash'),
     getAssetUrl = proxy.metaData.getAssetUrl,
-    SafeString = proxy.SafeString;
+    SafeString = proxy.SafeString,
+    hasMinFile;
 
 module.exports = function asset(path, options) {
-    var hasMinFile = _.get(options, 'hash.hasMinFile');
+    hasMinFile = _.get(options, 'hash.hasMinFile');
```

Comment:
Why was the variable turned into a global variable here?

Reply:
That's indeed odd, maybe remains of some testing I did back in May, now unneeded, sorry. Anyhow, all changes to the `{{asset}}` helper were reverted yesterday as suggested, and _asset.js_ is now an unmodified file.